cleanlab/object_detection/rank.py

```diff
@@ -66,6 +67,7 @@ def get_label_quality_scores(
     labels: List[Dict[str, Any]],
     predictions: List[np.ndarray],
     aggregation_weights: Optional[Dict[str, float]] = None,
+    overlapping_label_check: bool = True,
```

Comment:
why is this appearing before the *? It should be after as an optional argument that must be specified using keyword only.